<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>800</width>
      <height>621</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="emailLoginLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLoginLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>310</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailLoginInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLoginInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>510</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="registerWidnow">
     <widget class="QLineEdit" name="nameRegisterInput">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="surnameRegisterInput">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailRegisterInput">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordRegisterInput">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Password must contain:
- minimum 8 characters
- minimum 1 uppercase and 1 lowercase letter
- minimum 1 digit
- minimum 1 special charater (#?!@$%^&amp;*-)
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="registerToDatabaseButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>500</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="fullNameRegisterLabel">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Full Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="surnameRegisterLabel">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>150</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="emailRegisterLabel">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>200</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordRegisterLabel">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>240</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backToLoginPageButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>500</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Login Page</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="studentRegisterRadioButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Student</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="professorRegisterRadioButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Professor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="majorRegisterCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>400</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Aviation and Astronautics</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chemical Engineering and Technology</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chemical Technology</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Civil Engineering</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer Engineering</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Electrical Engineering</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Finance and Accounting</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Logistics</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modern Management</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="studentMajorRegisterLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>370</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Student Major / Professor Departament:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dateOfBirthRegisterLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Birth:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateOfBirthRegisterEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="studentMainPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>stud</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SMLogOutButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>530</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="professorMainPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prof</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PMLogOutButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>540</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
